packages/bot/src/services/tradeEngine/utils/broadcast.js

```diff
@@ -9,4 +9,8 @@ export const contractSettled = c => globalObserver.emit('contract.settled', c);
 export const info = i => globalObserver.emit('bot.info', i);
 
 // TODO use api time
-export const notify = (className, message) => globalObserver.emit('Notify', { className, message, position: 'right' });
+export const notify = (className, message) => globalObserver.emit('ui.log.notify', { className, message });
+
+export const log = (message) => globalObserver.emit('ui.log.success', { message });
+
+export const error = (message) => globalObserver.emit('ui.log.success', { message });
```

Comment:
- Should an error be `ui.log.success`? We have a `ui.log.error` event as well.
- Would it be an idea to prefix these generic functions? e.g. `notify_notifications`, `notify_system`, `notify_error`

Reply:
> Should an error be ui.log.success? We have a ui.log.error event as well.

ahh, i think i wrote it wrong

> Would it be an idea to prefix these generic functions? e.g. notify_notifications, notify_system, notify_error

I think is not necessarily since currently the only way we have to notify user is in journal?